plugins/dell-dock/fu-dell-dock-usb-usb4.c

```diff
@@ -510,7 +520,7 @@ fu_dell_dock_usb4_write_fw (FuDevice *device, FuFirmware *firmware,
 		return FALSE;
 
 	fu_device_add_flag (device, FWUPD_DEVICE_FLAG_NEEDS_ACTIVATION);
-	fu_device_set_version (device, fw_version);
+	fu_device_set_version(device, fw_version);
```

Comment:
@superm1 do you think it's okay to set a predicted version here as is yet authenticated? I know it's already in this way for a while. :)

Reply:
I think we've done this for some other devices too to intentionally prevent the same update getting staged twice while it's pending activation.  I don't see a problem with it, especially if you can't read needs activation on in between fwupdtool runs.